### ISSUE TITLE
make f-droid ready for using the new rust libraries

### ISSUE THREAD

**Issue by Dominik-K** [deltachat/deltachat-android#1111]
Hi,

there are no new APKs in F-Droid since months. `v0.510.1` (2019-07-05) is the current latest one. Fortunately the new update reminder (issue #885, PR https://github.com/deltachat/deltachat-android/pull/945) now shows up and hints us users to this issue :+1.

There are no Git tags with `v0.x.y` schema since `v0.510.1`  (only some occasional `preview-v0.x.y`) which trigger F-Droid's automated package creation.

**Comment by r10s** [comment]
wondering why the update reminder pops up when the app is installed from f-droid, afaik, exactly that should be avoided - as there is an update channel.

however, 0.510 is the last official version. we have worked the last months on porting the core from the c programming language to rust - [this nears completion now](https://github.com/orgs/deltachat/projects/15) - so it is correct that no new verisions pop up on f-droid - we've tagged he versions with the `preview-` to avoid generation on f-droid explicitly.

when we have a `v1.0.0` version, we will tag it as such and f-droid will theoretically start building it - however, this will require some more extra work for f-droid as the "old" build script is no longer working for rust, we have to do sth. as in https://github.com/deltachat/deltachat-android/blob/master/ndk-make.sh
@Dominik-K i do not know much about the f-droid build process - are you up for helping with that?
